plugins/woocommerce-blocks/tests/e2e/tests/product-filters-overlay-template-part/product-filters-overlay-template-part.block_theme.spec.ts

```diff
@@ -14,7 +14,7 @@ test.describe( 'Filters Overlay Template Part', () => {
 	} );
 
 	test( 'should be visible', async ( { page } ) => {
-		const block = page.getByText( 'Filters Overlay' );
+		const block = page.getByRole( 'button', { name: 'Filters Overlay 1' } );
```

Comment:
This tests for the template part area, not the actual template part.

Reply:
It's same. If the template part area exists, that means the template part was registered.